python/pyspark/sql/functions.py

```diff
@@ -2120,6 +2120,7 @@ def wrapper(*args):
                               else self.func.__class__.__module__)
         wrapper.func = self.func
         wrapper.returnType = self.returnType
+        wrapper._vectorized = self._vectorized
```

Comment:
nit: I think it should be `wrapper.vectorized` since wrapper is a function and it's no longer a private var

Reply:
Good call. I think it should not be underscore too.

Reply:
Fixed.